File: exercicios/R/1865/1865.r
input = file('stdin','r')

n = as.integer(readLines(input,n=1))

for(i in 1:n){
    v = strsplit(readLines(input,n=1), " ")

    if(v[[1]][1] == "Thor"){write("Y",'')}else{write("N",'')}
}

File: exercicios/JavaScript/1865/1865.js
var input = require('fs').readFileSync('stdin','utf8')
var lines = input.split('\n')

var x = lines.shift()

for(let i=0; i<x;i++){
    var td = lines.shift().split(' ')
    var st = td[0]

    if(st == 'Thor'){
        console.log('Y')
    }else{
        console.log('N')
    }
}